examples/QIR/Development/Development.csproj

```diff
@@ -64,7 +65,13 @@
       <Compile Include="Main.cs">
         <Visible>false</Visible>
       </Compile>
+      <_QirOutputFiles Include="$(QirOutputPath)**" />    
     </ItemGroup>
+    <!--
+    TODO: update Sdk version and uncomment these
```

Comment:
Does this `TODO` need to be resolved before merge?

Reply:
Yes, it should, but I need a full set of packages first.

Reply:
Removed; updated the package numbers for the two QIR examples.